Sources/Path.swift

```diff
@@ -47,66 +48,44 @@ public struct Path {
     /// The path of the program's current working directory.
     public static var current: Path {
         get {
-            return Path(FileManager.default.currentDirectoryPath)
+            return Path(Path.fileManager.currentDirectoryPath)
         }
         set {
-            FileManager.default.changeCurrentDirectoryPath(newValue._safeRawValue)
+            Path.fileManager.changeCurrentDirectoryPath(newValue._safeRawValue)
         }
     }
 
     /// The paths of the mounted volumes available.
     public static func volumes(_ options: FileManager.VolumeEnumerationOptions = []) -> [Path] {
-        let volumes = FileManager.default.mountedVolumeURLs(includingResourceValuesForKeys: nil,
+        let volumes = Path.fileManager.mountedVolumeURLs(includingResourceValuesForKeys: nil,
             options: options)
         return (volumes ?? []).flatMap { Path(url: $0) }
     }
 
     // MARK: - Properties
 
-    fileprivate var _fmWraper = _FMWrapper()
```

Comment:
Using a custom FileManager is a useful macOS feature, not really used for iOS
When you have a lot to do with files, you want to separate your `FileManagerDelegate` codes